src/Model/Profile.php

```diff
@@ -100,11 +108,15 @@ public static function load(App $a, $nickname, $profile = 0, $profiledata = [],
 			return;
 		}
 
-		if (empty($a->page['aside'])) {
+		// Not found in page array?
+		if (!isset($a->page['aside'])) {
+			// Then initialize it
+			/// @TODO Ugly workaround, find the root cause of this
```

Comment:
In some following commit in the same PR the array is now initialized.

Reply:
It should be initialized before the method is invoked. Let's find the root cause why it has not been initialized in the first place. This (current code) adds an `if()` that can be avoided reducing code complexity.

Reply:
This whole ```if``` thingy can be replaced, since we now initialize the array.

Reply:
What should I use then?

Reply:
It now is initialized in App::__construct

Reply:
What should I do then here?

Reply:
Remove the condition entirely please.